<launch>
    <node pkg="jetson_controller" type="control_jetson.py" name="jetson_controller" output="screen">
        <param name="interface" value="can0" />
        <param name="baudrate" value="250000" />
        <param name="odrive_dbc" value="$(find can_node)/odrive.dbc" />

        <remap from="/input/drive_command" to="/control/drive_command" />
    </node>
</launch>